cliquet/errors.py

```diff
@@ -72,6 +72,10 @@
 """
 
 
+class ReadOnlyError(Warning):
```

Comment:
Since you are using this as a warning, and you inherit from Warning, I would call it ReadOnlyWarning instead